src/_guides/language/effective-dart/style.md

```diff
@@ -450,114 +444,114 @@ but not for specifying generic types. Both `is` and `is!` are considered single
 binary operators. However, the `.` used to access members is not and should
 *not* have spaces around it.
 
-<div class="good">
+{:.good-style}
+<?code-excerpt "misc/lib/effective_dart/style_good.dart (bin-op)"?>
 {% prettify dart %}
 average = (a + b) / 2;
 largest = a > b ? a : b;
 if (obj is! SomeType) print('not SomeType');
-optional([parameter = defaultValue]) { ... }
```

Comment:
This section is about "DO place spaces around binary and ternary operators." The `optional(...)` seemed to add nothing that wasn't already illustrated by the previous examples. If this assumption is wrong, let me know.